app/Resources/views/base.html.twig

```diff
@@ -151,5 +151,7 @@
 
             <script src="{{ asset('js/app.js') }}"></script>
         {% endblock %}
+
+        <!-- Page rendered on {{ "now"|date('j/M/Y H:i:s') }} -->
```

Comment:
What about to use `localizeddate` filter here? Does it meaningless for public cached pages?